.changeset/dull-vans-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-cookie-banner": minor
+---
+
+[Added] - Page object supporting files and refactors
```

Comment:
Is a changeset required? I reckon we can get away without one seeing as there are no component changes

Reply:
Is not adding a page object support considered adding a feature? It does change how we test our components?

Reply:
Yeah, it's adding (test) functionality and improving the component, for sure, but I don't think it changes anything that is published to npm (currently?) so I don't think it makes sense to publish a new version.